src/main/java/uk/gov/hmcts/reform/bulkscan/orchestrator/model/ccd/ScannedDocument.java

```diff
@@ -26,6 +27,7 @@ public ScannedDocument(
         @JsonProperty("subtype") String subtype,
         @JsonProperty("scannedDate") LocalDateTime scannedDate,
         @JsonProperty("url") CcdDocument url,
+        @JsonProperty("deliveryDate") LocalDateTime deliveryDate,
```

Comment:
I presume this field is present in payload?

Reply:
it is present in `Envelope`.  I updated the pr to remove the field from the service bus model